docs/installation.rst

```diff
@@ -5,5 +5,15 @@ Install django-pgtransaction with::
 
     pip3 install django-pgtransaction
 
-After this, add ``pgtransaction`` to the ``INSTALLED_APPS``
-setting of your Django project.
+
+When used as a decorator, ``pgtransaction.atomic`` can be configured so
+that transactions are retried a certain number of times after encountering
+certain specified exceptions. One can configure custom defaults for these
+values in the their project's settings file:
+
+* To specify a default retry count, set ``PGTRANSACTION_RETRY`` to non-negative integer.
+  If not specified, ``PGTRANSACTION_RETRY`` defaults to 0.
+
+* To specify default exceptions to retry upon encountering, set ``PGTRANSACTION_RETRY_EXCEPTIONS``
+  to a tuple of exception classes. If not specified, ``PGTRANSACTION_RETRY_EXCEPTIONS``
+  defaults to ``(psycopg2.errors.SerializationFailure, psycopg2.errors.DeadlockDetected)``
```

Comment:
I would avoid putting setting information in the installation section and instead put it in its own section like [django-postgres-extra](https://django-postgres-extra.readthedocs.io/en/latest/installation.html) or [django-pgtrigger](https://django-pgtrigger.readthedocs.io/en/4.5.3/settings.html).

This library is also small enough that it may not need to call out settings again since the examples go over them